front/public/locales/fr/operational-studies.json

```
@@ -271,8 +271,12 @@
     "origin": "Origine",
     "pacedTrainUpdated": "Mission modifiée",
     "pacedTrains": {
+      "add": "Ajouter",
+      "addExtraOccurences": "Ajouter des occurences hors cadence",
```

Same in english. Also for the value, since both spelling seems to work in french I think it's easier if we spell occurrence with 2 'r' :)
```suggestion
      "addExtraOccurrences": "Ajouter des occurences hors cadence",
```

---

done